lib/server/push.api.js

```diff
@@ -321,9 +321,49 @@ function querySend(query, raixNotification) {
 
   let firebasePromises = [];
 
-  Push.appTokens.find(query).forEach(function (appToken) {
+  let userVetActiveMap = new Map();
+  let vetMap = new Map();
+
+  let userIdsSet = new Set();
+  let vetIdentifiersSet = new Set();
+
+  let appTokens = Push.appTokens.find(query).fetch();
```

Comment:
@vec12 You should create new function (extract this code), that will return appTokens array which will contain only tokens from active users. And use this function in both `push.api.js` and `push.api.legacy.js` files.